src/app/shared/components/lmt-autocomplete/lmt-autocomplete.component.ts

```diff
@@ -99,6 +112,23 @@ export class LmtAutocompleteComponent {
     );
   }
 
+  onChange: any = () => {}
+
+  onTouch: any = () => {}
+  
+
+  writeValue(obj: any): void {
+    this._values = obj;
+
+  }
```

Comment:
// allows Angular to register a function to call when the selected items changes.